packages/dashboard-frontend/src/services/workspace-client/devworkspace/devWorkspaceClient.ts

```diff
@@ -436,11 +440,10 @@ export class DevWorkspaceClient extends WorkspaceClient {
 
   /**
    * Initialize the given namespace
-   * @param namespace The namespace you want to initialize
-   * @returns If the namespace has been initialized
+   * @returns If the namespace has been provisioned
    */
-  async initializeNamespace(namespace: string): Promise<void> {
-    return DwCheApi.initializeNamespace(namespace);
+  async provisionKubernetesNamespace(): Promise<void> {
+    await this.workspaceClient.restApiClient.provisionKubernetesNamespace();
```

Comment:
I consider this should be an action in the InfrastructureNamespaces store. WDYT?

Reply:
I agree. I have fixed it.